calico-enterprise/visibility/visualize-traffic.mdx

```diff
@@ -2,6 +2,8 @@
 description: Learn the power of network sets.
 ---
 
+import ImageCaption from "../../../src/___new___/components/ImageCaption";
```

Comment:
Turns out the error is because of this import. As it happens, I've made this globally available, so no need to import. Try removing the import lines. That should resolved the errors.

Reply:
@ctauchen  it did not work with removing the import. However, I found the error was about a wrong link in the calico-cloud release note doc. I fixes that with my latest update to this PR.